examples/example-spring/src/test/kotlin/com/driver733/mapstructfluent/EmployeeMapperTest.kt

```diff
@@ -15,11 +13,19 @@ class EmployeeMapperTest(
     test("mapstruct impl") {
         val model = EmployeeModel("Alex")
 
-        model
-            .toEmployeeView()
-            .shouldBe(
-                mapper.toEmployeeView(model)
-            )
+        val actualEmployeeView = model.toEmployeeView()
+
+        actualEmployeeView shouldBe EmployeeView("Alex")
+        actualEmployeeView shouldBe mapper.toEmployeeView(model)
+    }
+
+    test("mapstruct additional arguments") {
+        val model = EmployeeModel("Alex")
+
+        val actualView = model.toEmployeeCompanyView("amazon")
```

Comment:
Rename to "actual" (I think that mentioning additional info in such small context is unnecessary)